### DISCUSSION TITLE
[v5] Error database migration: "column "column_name" of relation "table_name" already exists"

### DISCUSSION THREAD

**Discussion by prygun2** [adonisjs/core#2974] [Help]
Hi, guys!

I have a problem with my migration. When i tried to run it I received the error "column "column_name" of relation "table_name" already exists". At the same time, such a column has never existed and does not exist right now for sure.

Do you have any ideas?
![1](https://user-images.githubusercontent.com/19632389/127267033-7257ea53-b1ce-45cb-a446-84152d456f9a.png)
![2](https://user-images.githubusercontent.com/19632389/127267042-ae84d797-5396-4ecb-b530-fb80936ab171.png)
![3](https://user-images.githubusercontent.com/19632389/127267205-f0ccfe13-1c09-4d46-8fbe-1850ca97fbf6.png)

**Comment by thetutlage** [comment]
Can you share a screenshot of your orders table structure?

**Reply by prygun2** [reply]
I attached it in my post. If you mean someone else, let me know, please.
![4](https://user-images.githubusercontent.com/19632389/127268237-08a9cda9-7431-4772-887f-266152eedcd4.png)

**Comment by prygun2** [comment]
I ran the migration with raw sql and it worked
![5](https://user-images.githubusercontent.com/19632389/127282955-ad158856-8e55-4eaf-9a77-dbb437c692da.png)

**Reply by thetutlage** [reply]
Okay and can you try running the actual migration with the `--dry-run` flag and share the queries it generates?
